src/main/java/net/runelite/rsb/methods/Bank.java

```diff
@@ -826,12 +806,12 @@ public boolean withdraw(final int itemID, final int count) {
 				}
 				if (i == count) {
 					action = defaultAction;
-				}
-				else if (item.doAction("Withdraw-X")) {
+				} else if (item.doAction("Withdraw-X")) {
```

Comment:
I strongly prefer else ifs on newlines not stacked. This is inconsistent through the api though and I think the rest of the file is like this

Reply:
The entire project is like this and it's the default formatter behaviour of Intellij